firmware/screen/src/main.cpp

```
@@ -0,0 +1,400 @@
+}
+
+static void ws_tick() {
+  if (ws_state == WsState::Open) {
+    ws.poll();
+    if ((int32_t)(millis() - demo_next_at) >= 0) {
+      send_demo_action();
+      demo_next_at = millis() + DEMO_INTERVAL_MS;
+    }
+    return;
+  }
+  if (WiFi.status() != WL_CONNECTED) return;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)